lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -778,6 +778,11 @@
 
     // TYPE: upsell
     &.message-upsell {
+      padding-left: $baseline;
+      padding-top: $baseline;
+      padding-bottom: $baseline;
+      background-color: #F5F5F5;
```

Comment:
In edx-platform we can't use Paragon color names, but I wonder if there are other color variables with the same hex colors being used in other SCSS files i/o hard coding here

Reply:
I found this but it hasn't been updated since 2017 - https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/cms/static/sass/bootstrap/_variables.scss

Reply:
That seems to be the style colors for the cms, not the lms. I wasn't able to find #F5F5F5 associated with any other variables, but it's also not in the [Paragon colors section](https://paragon-edx.netlify.app/foundations/colors), so I'll double check with Eugene if that should be the proper gray hex color.

Reply:
I spoke to Eugene and we decided to use $light-300 for the gray background. I don't see it being used elsewhere in edx-platform or edx-themes so I'm going to hardcode the hex value here. It shouldn't be too big of a deal, since this page is slated to get replaced in the next few months.